### DISCUSSION TITLE
QUploader: allow supplying a custom upload function

### DISCUSSION THREAD

**Discussion by idc77** [quasarframework/quasar#16916] [Ideas / Proposals] [answered]
I like the QUploader component for the image preview, however I don't want it to handle file uploads, but use my own uploading function (connected to pinia and so on).
As far as I can tell there is no way to do that

**Comment by rstoenescu** [comment, marked as answer]
Hi,

There's a section especially for this in the docs: [Supporting other services](https://quasar.dev/vue-components/uploader#supporting-other-services)
